README.md

```diff
@@ -9,6 +9,15 @@ It follows the material design guidelines, and it is compatible starting from Ap
 ### Usage
 The usage is pretty straightforward. Add the tag into the XML layout, then use this snippet to populate with contents:
```

Comment:
I would prefer having this sentence split: "The usage is pretty straightforward. Add the tag into the XML layout:" here, and "then use this snippet to populate with contents:" below, after the `xml` snippet.